docs/installation/upgrading.rst

```diff
@@ -6,26 +6,80 @@ Upgrading KSQL
 Upgrade one KSQL server at a time (i.e. rolling restart). The remaining KSQL servers should have sufficient spare
 capacity to take over temporarily for unavailable, restarting servers.
 
+Upgrading to ksqlDB 5.5 from KSQL 5.4
+-------------------------------------
+
+#. Prepare a list of SQL statements for migration. One way is to dump the command
+   topic.
+
+   ```bash
+   ./bin/kafka-console-consumer --bootstrap-server localhost:9092 --topic _confluent-ksql-default__command_topic --from-beginning | jq ".statement" 
+   ```
+
+#. List your application's streams, tables, and types. Don't use the EXTENDED
+   option. Go through each list, and find the the last CREATE statement for each
+   stream/table/type name. Order the CREATE statements so they show the correct
+   dependencies.
+   
+#. Make other compatibility-related changes, as shown in the
+   `upgrade guide <https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/upgrading/#how-to-upgrade>`__.
+
+#. Stop the ``ksql`` service.
+
+#. Uninstall the v5.4 ksql rpm package.
+
+#. Install the v5.5 ksqldb rpm package.
+
+#. Copy the v5.4 configuration file to the v5.5 ksqldb configuration
+   folder, which is located at ``${CONFLUENT_HOME}/etc/ksqldb``:
+
+   ```bash
+   cp /etc/ksql/ksql-server.properties /etc/ksqldb/ksql-server.properties
+   ```
+
+#. Change the ``ksql.service.id`` in the property file, for example, from
+   ``default_`` to ``ksqldb_``.
+
+#. Create role bindings for the ``ksql`` service principal. For more information,
+   see :ref:`ksql-rbac`.
+
+#. Create three new role bindings:
```

Comment:
could be role bindings if you use rbac, acls if you use acls, or skip if you don't use authorization.
Role-bindings should be assigned to the ksql service principal.